### PR TITLE
[BREAKING] move attributes from db metrics to the registry

### PR DESCRIPTION
**Pull request by trisch-me**
Using tags for now (added moved attributes to the db-generic tag)

## Merge requirement checklist

* [x] [CONTRIBUTING.md](https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md) guidelines followed.
* [x] Change log entry added, according to the guidelines in [When to add a changelog entry](https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md#when-to-add-a-changelog-entry).
  * If your PR does not need a change log, start the PR title with `[chore]`
* [x] [schema-next.yaml](https://github.com/open-telemetry/semantic-conventions/blob/main/schema-next.yaml) updated with changes to existing conventions.

### CODE REVIEW

File: .chloggen/db_metrics.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+# your pull request title with [chore] or use the "Skip Changelog" label.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
+
+# The name of the area of concern in the attributes-registry, (e.g. http, cloud, db)
+component: db
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Rename `pool.name` to `db.pool.name` and `state` to `db.state`.
```
Comment (trask) [inline .chloggen/db_metrics.yaml:13]: do we need change log entry?

Comment (trask) [reply .chloggen/db_metrics.yaml:13]: oh, ignore me, I missed that this PR is also renaming attributes

**Review by trask — COMMENTED** [review]
maybe?

* `db.client.connections.pool.name`
* `db.client.connections.state`

they're long, but it matches the metric namespace where they're used, so they could theoretically(?) in the future be shortened back to `pool.name` and `state`

File: .chloggen/db_metrics.yaml (hunk 1 above)
Comment (lmolkova) [inline .chloggen/db_metrics.yaml:13]: I think `db.connection.pool.name` and ``db.connection.state` would be more descriptive

**Comment by trisch-me** [comment]
@trask @lmolkova thanks for suggestion, I have updated fields to include the prefix

**Review by lmolkova — APPROVED** [review]

**Review by joaopgrassi — APPROVED** [review]

**Review by pyohannes — APPROVED** [review]

**Comment by joaopgrassi** [comment]
@trask do you want to take a look at this before we merge (since you left a review).

**Review by trask — APPROVED** [review]

**Pull request merged**